apis/Google.Bigquery.V2/Google.Bigquery.V2/BigqueryClient.Queries.cs

```diff
@@ -174,5 +176,180 @@ public virtual BigqueryQueryJob PollQueryUntilCompleted(JobReference jobReferenc
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Asynchronously executes a query.
+        /// </summary>
+        /// <param name="sql">The SQL query. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>A task representing the asynchronous operation. When complete, the result is
+        /// the <see cref="BigqueryQueryJob"/> representing the query.</returns>
+        public virtual Task<BigqueryQueryJob> ExecuteQueryAsync(string sql, ExecuteQueryOptions options = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Asynchronously executes a command. This overload allows query parameterization, and is preferred over
+        /// <see cref="ExecuteQueryAsync(string, ExecuteQueryOptions,CancellationToken)"/> when values need to be passed in.
+        /// </summary>
+        /// <param name="command">The command to execute. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>A task representing the asynchronous operation. When complete, the result is
+        /// the <see cref="BigqueryQueryJob"/> representing the query.</returns>
+        public virtual Task<BigqueryQueryJob> ExecuteQueryAsync(BigqueryCommand command, ExecuteQueryOptions options = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Asynchronously creates a job for a SQL query, with more facilities than <see cref="ExecuteQueryAsync(string, ExecuteQueryOptions,CancellationToken)"/>, including the option
+        /// to store the results in a persistent table.
+        /// </summary>
+        /// <param name="sql">The SQL query. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <returns>The query job created. Use <see cref="GetQueryJobAsync(JobReference,GetQueryResultsOptions,CancellationToken)"/> to retrieve
```

Comment:
I think this (and the overload below) should start with "A task representing the asynchronous operation. When complete, the result is"